src/ActiveMarker.tsx

```diff
@@ -1,26 +1,35 @@
-import React from 'react';
+import React, { MouseEventHandler } from 'react';
 import styled from 'styled-components';
 
 interface ActiveMarkerProps {
   activeMarker?: Marker;
   tracks: TrackWithMarkers[];
+  setActiveMarker: (marker: Marker) => void;
 }
 
 interface KeyframeBlockProps {
   title: string;
   keyframe: TrackKeyframe;
   percentage: number;
-  active?: boolean;
+  activate?: () => void;
 }
 
 const ActiveMarkerWrapper = styled.div`
   flex-basis: 20vw;
   padding: 20px;
 `;
 
-const KeyframeBlock = ({ title, keyframe, percentage }: KeyframeBlockProps) => {
+const KeyframeBlock = ({
+  title,
+  keyframe,
+  percentage,
+  activate,
+}: KeyframeBlockProps) => {
+  const onClick: MouseEventHandler<HTMLDivElement> = () => {
+    activate && activate();
+  };
   return (
-    <div>
+    <div onClick={onClick}>
```

Comment:
should this be a button instead of a div?

Reply:
Good shout, yes it probably should.